Packs/Unit42byPaloAltoNetworks/Integrations/Unit42Intelligence/Unit42Intelligence.yml

```
@@ -0,0 +1,316 @@
+category: Data Enrichment & Threat Intelligence
+sectionOrder:
+- Connect
+- Collect
+commonfields:
+  id: Unit 42 Intelligence
+  version: -1
+configuration:
+- display: Server URL
+  defaultvalue: https://xdr-tas-api.crtx.us.paloaltonetworks.com/
+  name: url
+  required: true
+  type: 0
+  section: Connect
+  additionalinfo: The base URL for the Unit 42 Intelligence API
+- display: Source Reliability
+  name: integration_reliability
+  additionalinfo: Reliability of the source providing the intelligence data.
+  type: 15
+  options:
+  - A++ - 1st party feed and enrichment
+  - A+ - 3rd party enrichment
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  defaultvalue: A++ - ?????
+  required: false
+  section: Collect
+  advanced: true
+- display: Create relationships
+  name: create_relationships
+  additionalinfo: Create relationships between indicators and threat objects
+  type: 8
+  defaultvalue: 'true'
+  required: false
+  section: Collect
+  advanced: true
+- display: Create threat objects as separate indicators
+  name: create_threat_object_indicators
+  additionalinfo: Whether to create threat objects (malware families, actors, campaigns, etc.) as separate XSOAR indicators
+  type: 8
+  defaultvalue: 'false'
```

```suggestion
  defaultvalue: 'true'
```
PM is asking that we change this to true.  Any reason we shouldn't? CC @Ni-Knight

---

@Ni-Knight seems to not mind: https://panw-global.slack.com/archives/C091N069V70/p1758449224646729?thread_ts=1758226749.301979&cid=C091N069V70